README.md

```diff
@@ -1063,48 +1063,59 @@ Default: `false`.
-  Set this to true to change the behavior of the `login_name` parameter so that only logins exactly matching one of the provided login_names is returned in the result set.
-  This is an optional parameter which will accept either `true` or `false`. It's default value is false.
+Set this to true to change the behavior of the `login_name` parameter so that only logins exactly matching one of the provided login_names is returned in the result set.
+
+This is an optional parameter which accepts either `true` or `false`. It's default value is false.
 
 * **detailed**
 
-  This parameter causes the task to return a more detailed level of information for each login.
-  By default the list of properties returned about each login is: `Name`,`isDisabled`,`isLocked`,`IsPasswordExpired`,`CreateDate`,`DateLastModified`.
-  Setting this parameter to true adds in the following properties: `DefaultDatabase`,`DenyWindowsLogin`,`HasAccess`,`ID`,`IsSystemObject`,`Language`,`LanguageAlias`,`,LoginType`,`MustChangePassword`,`PasswordExpirationEnabled`,`PasswordHashAlgorithm`,`PasswordPolicyEnforced`,`SQLSID`,`ADSid`,`WindowsLoginAccessType`,`UserData`,`State`,`IsDesignMode`,`InstanceName`
+This parameter causes the task to return a more detailed information for each login.
 
-> **A note about `SQLSID` and `ADSID`.**
+By default, the list of properties returned about each login is: 
+
+`Name`,`isDisabled`,`isLocked`,`IsPasswordExpired`,`CreateDate`,`DateLastModified`.
+
+Setting this parameter to true adds in the following properties: 
+
+`DefaultDatabase`,`DenyWindowsLogin`,`HasAccess`,`ID`,`IsSystemObject`,`Language`,`LanguageAlias`,`,LoginType`,`MustChangePassword`,`PasswordExpirationEnabled`,`PasswordHashAlgorithm`,`PasswordPolicyEnforced`,`SQLSID`,`ADSid`,`WindowsLoginAccessType`,`UserData`,`State`,`IsDesignMode`,`InstanceName`
+
+> **Note about `SQLSID` and `ADSID`.**
 >
 > The `SQLSID` property is this module's property name for the binary representation of a SID that SQLServer keeps internally in tables like `sys.server_principals`.
 > It will look like `0x01` for accounts like `sa`, or something longer like `0x0106000000000009010000005FB6DAC7F7DB546D706711B128B5063888B01770` for other accounts.
-> This `sid` does not look like a normal `sid` you might see outside of SQLServer, but it is returned as part of the detailed information to make it easier to correlate the logins returned by this module and query results from SQLServer.
-> The `ADSID` property is a more normal looking `sid` you might get from PowerShell Active Directory query tools.
-> It is a direct translation of that `SQLSID` into the Microsoft string `sid` form and will look something like `S-1-5-80-1402415987-66678372-3059512406-1823130485-2345841878`.
+> This `sid` does not look like a normal `sid` you might see outside of SQLServer, but instead it is returned as part of the detailed information to make it easier to correlate the logins returned by this module and query results from SQLServer.
```

Comment:
I don't think the word `instead` is needed here. The fact that it's being returned in the detailed information is not an alternative. The `but` here is to explain why it is that I'm returning data that may not look familiar to the user and to explain why.

Reply:
@RandomNoun7 I updated this.